Cython/Includes/cpython/time.pxd

```diff
@@ -0,0 +1,39 @@
+"""
+Cython implementation of (parts of) the standard library time module.
+"""
+
+from libc.stdint cimport int64_t
+
+ctypedef int64_t _PyTime_t
```

Comment:
This is defined in `pytime.h` (included from `Python.h`), so this should be declared `extern` (together with the functions below).